scala-steward.conf

```diff
@@ -19,3 +19,8 @@ dependencyOverrides = [
     pullRequests = { frequency = "30 days" }
    }
 ]
+
+# Ignore Akka updates following licence changes. Note, we hope to provide better
+# solutions to this going forward; this is a crude sticking-plaster to avoid us
+# sleepwalking into lots of fees.
+updates.ignore = [ { groupId = "com.typesafe.akka" }, {groupId = "com.lightbend.akka"} ]
```

Comment:
Looks great to me - just to observe on those two group ids:

* `com.typesafe.akka` is used for artifacts like [`"com.typesafe.akka" %% "akka-http"`](https://search.maven.org/artifact/com.typesafe.akka/akka-http_2.13) ([BSL](https://github.com/akka/akka-http/blob/main/LICENSE.txt))
* `com.lightbend.akka` is used for artifacts like [`"com.lightbend.akka" %% "akka-stream-alpakka-dynamodb_2.13"`](https://search.maven.org/artifact/com.lightbend.akka/akka-stream-alpakka-dynamodb_2.13) ([BSL](https://github.com/akka/alpakka/blob/main/LICENSE)), but it's also used as a _prefix_ for group ids like `com.lightbend.akka.grpc` and `com.lightbend.akka.management`, and I think [Scala Steward will only filter on full group ids](https://github.com/scala-steward-org/scala-steward/blob/89c66e36a42425857250b17be0b1be55aa0fad02/modules/core/src/main/scala/org/scalasteward/core/repoconfig/UpdatePattern.scala#L43)

...so it might eventually become necessary to add a few more prefixed group ids in, but this is a good for now 👍